public/video-ui/src/reducers/configReducer.js

```diff
@@ -0,0 +1,11 @@
+export default function config(state = false, action) {
+  switch (action.type) {
+
+    case 'CONFIG_RECEIVED':
+        return action.config || false;
```

Comment:
This is just how the video was done, I agree though, we should return an empty object, then at least it's be undefined and not an error